app/src/js/utils/table-config/granules.js

```diff
@@ -112,6 +114,33 @@ export const errorTableColumns = [
   }
 ];
 
+export const defaultWorkflowMeta = JSON.stringify({ meta: {} }, null, 2);
+
+export const executeDialog = (config) => (
+  <div>
+    <SimpleDropdown
+      key={config.label}
+      label={config.label.toUpperCase()}
+      value={config.value}
+      options={config.options}
+      id={config.label}
+      onChange={config.selectHandler}
+    />
+    <Collapsible trigger='Add Custom Workflow Meta' triggerWhenOpen='Collapse'>
+      <form>
+        <TextArea
+          value={config.initialMeta}
+          id={`${config.label}-meta-text-area`}
+          onChange={config.metaHandler}
```

Comment:
I noticed when you completely clear out the meta text, the value changes to "workflow-meta-text area". Looks like this is only happening in places where it's directly using the useState setters, since there's no wrapping function to handle the id that's passed (so `Collections/granules` and `Granules/list`. Might be easier to just handle the id once right here to cover all cases with something like this:

```suggestion
          onChange={(id, value) => config.metaHandler(value)}
```

Up to you though on how you want to handle it!

Reply:
That looks good. I pushed the requested updates.